.github/copilot-instructions.md

```diff
@@ -0,0 +1,26 @@
+## Coding Style
+
+-   Use spaces for indentation with four-spaces per level, unless it is a csproj file, then use two-spaces per level.
+-   For integrations use the following namespace rules:
+    -   If the integration is a hosting integration (starts with CommunityToolkit.Aspire.Hosting), extension methods should be placed in `Aspire.Hosting`.
+    -   If the integration is a client integration (starts with CommunityToolkit.Aspire), extension methods should be placed in `Microsoft.Extensions.Hosting`.
+    -   Use file-scoped namespaces.
+-   All public methods require doc comments
+-   Prefer type declarations over `var`
```

Comment:
This seems contradicting to the code sample be shown below, as it uses `var`. Reworded a bit.

```suggestion
-   Prefer type declarations over `var` when the type isn't obvious.
```

Reply:
Do as I say and not as I do 🤣